CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 All notable changes to this project are documented in this file.
 See also [TiDB Changelog](https://github.com/pingcap/tidb/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.1.6]
+- Fix the `StoreNotMatch` issue caused by decoding protobuf error in some cases [#4303](https://github.com/tikv/tikv/pull/4303)
+- Improve import speed by increasing default region-split-size to 512 MiB [#4347](https://github.com/tikv/tikv/pull/4347)
+- Fix OOM issue by storing the intermediate SST files on disk instead of memory [#4348](https://github.com/tikv/tikv/pull/4348)
+- Restrict memory usage by RocksDB [#4350](https://github.com/tikv/tikv/pull/4350)
+- Fix the issue about scatter Regions when Regions are still imbalance [#4352](https://github.com/tikv/tikv/pull/4352)
```

Comment:
the description is wrong, should be `Fix the issue that scattering Region doesn't take effect`